AddressParser/AddressParser.cs

```diff
@@ -928,6 +937,33 @@ public AddressParseResult ParseAddress(string input, bool normalize)
             return new AddressParseResult(extracted);
         }
 
+        public AddressParseResult ParseAddressLine(string input, bool normalize)
```

Comment:
Sorry, I only saw one suggestion and did not  know that was the FINAL design or not up for discussion.  Sure I can change it, but sort of harsh...